### DISCUSSION TITLE
How to quit app with CTRL+C ?

### DISCUSSION THREAD

**Discussion by lcmgh** [ratatui/ratatui#226] [Discussion]
I wonder how I can properly quit my application when user presses ctrl+c. I can't find such an event in `crossterm::event::KeyCode`. Using https://github.com/Detegr/rust-ctrlc has not worked.

In this example it does not work as well `cargo run --example table`

**Comment by orhun** [comment]
You can use the combination of [`KeyCode::Char('c')`](https://docs.rs/crossterm/latest/crossterm/event/enum.KeyCode.html#variant.Char) and [`KeyModifiers::CONTROL`](https://docs.rs/crossterm/latest/crossterm/event/struct.KeyModifiers.html#associatedconstant.CONTROL)

**Reply by lcmgh** [reply]
Thank you, that worked!
